charts/turbinia/templates/secret.yaml

```diff
@@ -7,17 +6,30 @@ metadata:
   labels:
     {{- include "turbinia.labels" . | nindent 4 }}
 stringData:
-  turbinia-secret: |
+  {{- if .Values.oauth2proxy.enabled }}
+  dftimewolf-secret: |
     {
       installed:
         {
-            "client_id": {{ .Values.oauth2proxy.configuration.clientID | quote }},
-            "client_secret": {{ .Values.oauth2proxy.configuration.clientSecret | quote }},
-            "redirect_uris": [{{ .Values.oauth2proxy.configuration.redirectUrl | quote }}],
-            "project_id": {{ .Values.gcp.projectID | quote }},
+            "client_id": "<NATIVE_CLIENT_ID>",
+            "client_secret": "<NATIVE_CLIENT_SECRET>",
+            "redirect_uris": [ "<https://<DOMAIN>.com/oauth2/callback>" ],
+            "project_id": "<GCP_PROJECT_ID>",
```

Comment:
Maybe I'm missing something here about how helm works, but wouldn't this change remove the option to configure the secret during the set-up via the values? Or does it require the user to go into this file and replace those values manually?

Reply:
Thank you for taking a look @jkppr !!

RE the secret, good call out but yes it will require the user to go into this file and replace those values manually (which we note in the NOTES.txt file). I realized the previous method did not work either because it was configuring the web oauth client not the native oauth client.

I went ahead and updated this so the values that are available are replaced and the user will only have to configure the native client secret which there will be a note telling the user to do so in the NOTES.txt file. 

Hoping to make this easier in the future by either providing a separate script to generate all appropriate configs or by having some sort of global controller with the CLIs/configs pre-installed